src/status_im2/contexts/chat/composer/style.cljs

```diff
@@ -77,6 +77,10 @@
                                   (if link-previews? constants/links-container-height 0)
                                   (if (seq images) constants/images-container-height 0))
           :padding-bottom      (when @maximized? 0)}))
+
+(def container
+  {:z-index 5})
+
```

Comment:
@alwx the newly introduced issues are from the `z-index`. You can revert it for now and I will look into it in another issue to not block other fixes

Reply:
Ok, will leave it up to you to fix the positioning